### PR TITLE
Added package.json

### PR DESCRIPTION
**Pull request by bennyn**
Because we want to use this project in Node.js, we need to declare a `package.json` file to make it available via https://www.npmjs.com.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Protocol definition for generic messages.",
+  "license": "GPL-3.0",
+  "main": "./proto",
+  "name": "wire-webapp-protocol-message",
```
Comment (lipis) [inline package.json]: I can see it's alphabetical.. but it still feels that the name should be first maybe?!

Comment (lipis) [inline package.json]: But on the other hand.. maybe not :)

**Comment by bennyn** [comment]
Can this be merged?

**Comment by marcoconti83** [comment]
Do you guys still want this? If yes, just merge it, no objection on our side

**Pull request merged**